rqd/rqd/rqconstants.py

```diff
@@ -52,6 +52,8 @@
 RQD_GRPC_MAX_WORKERS = 10
 RQD_GRPC_PORT = 8444
 RQD_GRPC_SLEEP = 60 * 60 * 24
+RQD_GRPC_CONNECTION_ATTEMPT_SLEEP = 15
```

Comment:
Could you add a unit to the var name? `RQD_GRPC_CONNECTION_ATTEMPT_SLEEP_SEC` maybe?

Reply:
Var name above it could use it as well, but maybe that's out of scope here.

Reply:
Added "_SEC" to both variables.